main.go

```diff
@@ -211,15 +211,17 @@ func WaitDaemonsetReady(namespace, name string, timeout time.Duration) error {
 			return fmt.Errorf("failed to get daemonset, err: %s", err)
 		}
 
-		if daemonSet.Status.DesiredNumberScheduled != nodesCount {
-			return fmt.Errorf("daemonset DesiredNumberScheduled not equal to number of nodes:%d, please instantiate debug pods on all nodes", nodesCount)
+		if daemonSet.Status.DesiredNumberScheduled == nodesCount {
+			logrus.Infof("Waiting for (%d) debug pods to be ready: %+v", nodesCount, daemonSet.Status)
```

Comment:
The message needs to be changed. There is no "debug" pod. This is a generic daemonset used in many places. My suggestion would be to use daemonset <<name>> pods instead of debug pods.

Reply:
Done.